tests/stdlib/thttpclient_standalone.nim

```diff
@@ -35,20 +52,20 @@ block: # bug #14794 (And test for presence of content-length header when using p
       killServer = true
       asyncCheck req.respond(Http200, "OK")
 
-    var server = newAsyncHttpServer()
-    server.listen(Port(5556))
     while not killServer:
       if server.shouldAcceptRequest():
         await server.acceptRequest(cb)
       else:
         poll()
 
-  proc runClient() {.async.} =
+  proc runClient(port: Port) {.async.} =
     let c = newAsyncHttpClient()
     var data = newMultipartData()
     data.add("file.txt", "This is intended to be an example text file.\r\nThis would be the second line.\r\n")
-    let r = await c.postContent("http://127.0.0.1:5556", multipart = data)
+    discard await c.postContent("http://127.0.0.1:" & $(uint16)port, multipart = data)
     c.close()
 
-  asyncCheck startServer()
-  waitFor runClient()
+  var server = startServer()
+  var port = server.getPort()
```

Comment:
let
ditto in other places

Reply:
Thanks for the input.  I've changed as suggested.